README.md

```diff
@@ -122,6 +122,36 @@ your container. This will allow you to login to the root account remotely. Local
 connections will still not require a password.
 
 
+Usage on Atomic host
+---------------------------------
+Systems derived from projectatomic.io usually include `atomic` command that is
+used to run containers except other tasks.
+
+To install a new service `mysql1` based on this image on such a system, run:
+
+```
+$ atomic install -n mysql1 --opt2='-e MYSQL_USER=user` -e MYSQL_PASSWORD=secretpass -e MYSQL_DATABASE=db1 -p 3306:3306' THIS_IMAGE
```

Comment:
"THIS_IMAGE" would be rhscl/mysql-56-centos7 for example?  might be better to provide a concrete example or something that at least looks more like a docker pull spec.